lib/pages/settings/view/settings_page.dart

```diff
@@ -23,6 +24,20 @@ class SettingsPage extends StatefulWidget {
 }
 
 class _SettingsPageState extends State<SettingsPage> {
+  PackageInfo _packageInfo = PackageInfo(
+    version: 'Unknown',
+  );
+
+  Future<void> _initPackageInfo() async {
+    // Package_info_plus is not compatible with Flutter_Test
```

Comment:
What I said vs what you did 😥 where's the issue link?
Also, don't use random uppercase in package names, they're all lowercase.